tests/cases/fourslash/codeFixImportNonExportedMember3.ts

```diff
@@ -0,0 +1,27 @@
+/// <reference path='fourslash.ts' />
+// @Filename: /a.ts
+////let a = 1, b = 2, c = 3;
+////let d = 4;
+////export function whatever() {
+////}
+////export { d }
+
+// @Filename: /b.ts
+////import { a, d } from "./a"
+
+goTo.file("/b.ts");
+verify.codeFixAvailable([
+    { description: `Export 'a' from module './a'` },
+    { description: `Remove import from './a'` },
+]);
+verify.codeFix({
+    index: 0,
+    description: `Export 'a' from module './a'`,
+    newFileContent: {
+        '/a.ts': `let a = 1, b = 2, c = 3;
+let d = 4;
+export function whatever() {
+}
+export { d, a };`

```

Comment:
The current solution is append to tail, if we want to reorder according to the declare order, the situation may complex and produces additional maintenance burden.

Reply:
I see two options:
* Try to reverse the order with some samples, and maybe they'd be generated in the source order.
* If not, then maybe sort the names alphabetically -- that would be sensible too.

I think that it's generally bad to leave the order unspecified since that means unreliable results.

Reply:
@elibarzilay Thanks for your advise, I will take the second option.